src/chatops/custom-runner/fly.md

```diff
@@ -0,0 +1,64 @@
+---
+uid: runner-on-fly
+Title: "Tutorial: Deploying a runner to Fly.io"
+---
+
+## Tutorial: Deploying a runner to Fly.io
+
+[Fly.io](https://fly.io) is a platform for quickly launching applications all over the world.
+It's a great platform for hosting Abbot skill runners, since you can quickly launch any Docker image you want.
+
+To begin, make sure you've signed up for an account at [Fly.io](https://fly.io).
+Download the [Fly CLI](https://fly.io/docs/hands-on/install-flyctl/) and log in to it.
+
+Create a new Fly application by moving to an empty folder on your machine and running the following command:
+
+```
+> flyctl launch
+```
+
+Enter any app name you'd like, select the organization and region (we recommend `sea`).
```

Comment:
Why do we recommend `sea`?

Reply:
Because that's where Abbot is (approximately), I suppose we could test to find the best region by latency. Alternatively, I could just drop the recommendation altogether.

Reply:
Or include why we recommend it.